.github/workflows/validate.yml

```diff
@@ -210,6 +210,7 @@ jobs:
         run: sh validate.sh $FLAGS -s cli-tests
 
       - name: Validate cli-suite
+        if: runner.os != 'Windows'
```

Comment:
See line 177 for how to do this properly; I think that won't work as is (`runner.os` won't be expanded).

Reply:
Okay, actually it looks like it did work (skipped `cli-suite` only on Windows) but we probably want to be consistent.

Reply:
Line 177 is a part of a bash expression, and I act on the yaml level. So, I'm not sure why they should be consistent necessarily: they're pretty different notations...